tests/test_grpc.py

```diff
@@ -93,6 +93,32 @@ def test_download_missing_file(mapdl, tmpdir):
         mapdl.download('__notafile__', target)
 
 
+def test_grpc_custom_ip():
+    from ansys.mapdl.core import launch_mapdl
+    
+    ip='127.0.0.2'
+    mapdl = launch_mapdl(ip=ip)
+
+    # Check the ip
+    # print(mapdl._ip)
+
+    # Check it also in the output file.
+    # print(mapdl.directory)
+
+    output_file = mapdl._download_as_raw('.__tmp__.out')
+    output = output_file.decode().splitlines()
+
+    # Catching the line where the IP of the gRPC server is specified.
+    for each_line in output:
+        if 'Server listening on' in each_line: 
+            line = each_line
+            print(each_line)
+
+    mapdl.exit()
+
+    assert ip in line
+
```

Comment:
Add a check here to make sure the temporary local ip file was removed.  Removing garbage files isn't a big deal as MAPDL creates a ton of temp files, but it's nice to not add another one.